### PR TITLE
Add dotnet7 preview6 sdk

### PR DESCRIPTION
**Pull request by william-msft**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1554640
- [ ] Tests are included and/or updated for code changes.
~- [ ] Proper license headers are included in each file.~

### CODE REVIEW

File: src/BuildScriptGenerator/PythonVersions.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public static class PythonVersions
         public const string Python37Version = "3.7.12";
         public const string Python38Version = "3.8.12";
         public const string Python39Version = "3.9.7";
-        public const string Python310Version = "3.10.4";
+        public const string Python310Version = "3.10.0";
```
Comment (qianz2) [inline src/BuildScriptGenerator/PythonVersions.cs:16]: This line needs to be reverted as currently we have 3.10.4 as default version

File: src/BuildScriptGenerator/DotNetCore/DotNetCoreSdkVersions.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ public static class DotNetCoreSdkVersions
         public const string DotNet50SdkVersion = "5.0.408";
         public const string DotNet60SdkVersion = "6.0.301";
         public const string DotNet60RtmSdkVersion = "6-0-100-rtm-21527-11";
-        public const string DotNet70SdkVersion = "7.0.100-preview.5.22307.18";
+        public const string DotNet70SdkVersion = "7.0.100-preview.6.22352.1";
```
Comment (qianz2) [inline src/BuildScriptGenerator/DotNetCore/DotNetCoreSdkVersions.cs:18]: Can we add a test to check if versions are added to the storage account?

File: src/BuildScriptGenerator/PythonVersions.cs (hunk 1 above)
Comment (william-msft) [reply src/BuildScriptGenerator/PythonVersions.cs]: Thanks for catching this! 😃

**Review by cormacpayne — DISMISSED** [review]
:shipit:

**Review by qianz2 — DISMISSED** [review]

File: src/BuildScriptGenerator/DotNetCore/DotNetCoreSdkVersions.cs (hunk 2 above)
Comment (william-msft) [reply src/BuildScriptGenerator/DotNetCore/DotNetCoreSdkVersions.cs:18]: Test added and tested locally:
image

**Review by qianz2 — DISMISSED** [review]

**Review by qianz2 — DISMISSED** [review]

**Review by cormacpayne — APPROVED** [review]

**Pull request merged**